.golangci.yml

```diff
@@ -11,11 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+version: "2"
```

Comment:
Please keep an empty line between boilerplate and actual file contents.

Reply:
done!